packages/expo-cli/src/commands/build/AndroidBuilder.ts

```diff
@@ -60,13 +60,15 @@ export default class AndroidBuilder extends BaseBuilder {
   }
 
   async collectAndValidateCredentials(): Promise<void> {
-    const ctx = new Context();
+    const nonInteractive = this.options.parent?.nonInteractive === true;
```

Comment:
remove this part from here and other places where it's used
`this.options.parent?.nonInteractive === true`